--- base/CMakeLists.txt.org	2013-03-14 22:07:33.569300876 +0100
+++ base/CMakeLists.txt	2013-03-14 22:09:39.422327063 +0100
@@ -27,7 +27,7 @@
 set (HARDENING_FLAGS            "-Wformat -Wformat-security -O2 -D_FORTIFY_SOURCE=2 -fstack-protector -Wl,-z,relro -Wl,-z,now")
 
 if (NOT MINGW)
-  set (CMAKE_C_FLAGS_DEBUG        "${CMAKE_C_FLAGS_DEBUG} -Werror")
+  set (CMAKE_C_FLAGS_DEBUG        "${CMAKE_C_FLAGS_DEBUG}")
   # FIXME: certain flags should be defined on the global level,
   # because they change the API (e.g. _FILE_OFFSET_BITS).
   set (CMAKE_C_FLAGS              "${CMAKE_C_FLAGS} ${HARDENING_FLAGS} -D_FILE_OFFSET_BITS=64 -Wall -fPIC")
