--- omp/CMakeLists.txt.org	2013-03-14 22:08:27.628631198 +0100
+++ omp/CMakeLists.txt	2013-03-14 22:09:39.425658712 +0100
@@ -26,7 +26,7 @@
 
 set (HARDENING_FLAGS            "-Wformat -Wformat-security -O2 -D_FORTIFY_SOURCE=2 -fstack-protector -Wl,-z,relro -Wl,-z,now")
 
-set (CMAKE_C_FLAGS_DEBUG        "${CMAKE_C_FLAGS_DEBUG} -Werror")
+set (CMAKE_C_FLAGS_DEBUG        "${CMAKE_C_FLAGS_DEBUG}")
 set (CMAKE_C_FLAGS              "${CMAKE_C_FLAGS} ${HARDENING_FLAGS} -Wall")
 
 set (FILES xml.c omp.c)
